Linux/flash

```diff
@@ -131,15 +131,10 @@ fi
 
 case "${OSTYPE}" in
   darwin*)
-    echo This version does not support Mac.
-    echo Download Mac version from https://github.com/hypriot/flash instead.
-    exit 11
-    ;;
-  Linux)
-    ;;
-  linux)
+    source $(dirname $0)/darwin.func
```

Comment:
Using external script is good for now. 
Later on it would be great to have these functions built-in again so we can just `curl` the single script.

Reply:
Good point! I have not thought about that one.